packages/esm-patient-chart-app/src/encounter-list/components/encounter-list-tabs.component.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo } from 'react';
+import { useConfig, usePatient, useVisit } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+import { Tabs, Tab, TabList, TabPanels, TabPanel } from '@carbon/react';
+import { EncounterList } from './encounter-list.component';
+import { getMenuItemTabsConfiguration } from '../utils/encounter-list-config-builder';
+import styles from './encounter-list-tabs.scss';
+import { filter } from '../utils/helpers';
+import { type Encounter } from '../types';
+
+interface EncounterListTabsComponentProps {
+  patientUuid: string;
+}
+
+const EncounterListTabsComponent: React.FC<EncounterListTabsComponentProps> = ({ patientUuid }) => {
+  const config = useConfig();
+  const { tabDefinitions = [] } = config;
+  const { t } = useTranslation();
+  const tabsConfig = getMenuItemTabsConfiguration(tabDefinitions);
```

Comment:
So, in `getObsFromEncounter()`, we still have multiple references like this:

```ts
esmPatientChartSchema.falseConceptUuid._default
```

As I've pointed out a few times, this is hard-coding the values that are defined in the schema. To address this, we need to ensure that the `trueConceptUuid`, `falseConceptUuid`, and `otherConceptUuid` are actually resolved from the config. To do that, we need to pass in the config from somewhere, like here where we get it from `useConfig()`.

Basically, if we can pass the config as a property here, then in `getMenuItemTabsConfiguration()` we can associate the config with each tab definition, and then we should be able to use it where necessary.

Reply:
Resulted to passing it as a prop in the function since they are not components.